test/functional/ui/embed_spec.lua

```diff
@@ -133,3 +136,60 @@ describe('--embed UI', function()
     ]]}
   end)
 end)
+
+describe('--embed --listen UI', function()
+  it('waits for connection on listening address', function()
```

Comment:
Hmm it seems to be the same hang as the one in `swapfile_preserve_recover_spec.lua`: any RPC request hangs for the child session. I think this should be skipped on Windows for now then.
```suggestion
  it('waits for connection on listening address', function()
    helpers.skip(helpers.is_os('win'))
```